### ISSUE TITLE
Usage information should be in scripts/README.md

### ISSUE THREAD

**Issue by rufuspollock** [datasets/inflation#1]
Can we move this info to scripts/README.md https://github.com/datasets/inflation#usage-of-inflation2datapackagepy

**Comment by zelima** [comment]
@rgrp I moved usage information in to new README.md in scripts folder. 
The README.md in inflation folder is left, but without usage information?
Correct, or i should move entire README.md into scripts folder?

**Comment by rufuspollock** [comment]
@zelima yes that's right. Sometimes if you want you can leave a heading called "Preparation" and say under it something like:

"See scripts/README.md"

BTW I assume you have read through http://data.okfn.org/doc/publish-faq#readme

**Comment by zelima** [comment]
I'll read this more carefully now.
Will modify READMY.md after we deal with script problem.

**Comment by zelima** [comment]
@rgrp please review https://github.com/datasets/inflation/blob/master/scripts/README.md
